common/vision/lasr_vision_deepface/CMakeLists.txt

```diff
@@ -163,6 +163,11 @@ catkin_install_python(PROGRAMS
   scripts/create_dataset
   examples/relay
   examples/greet
+  examples/greet_and_identify
+  nodes/actions/get_name
+  nodes/actions/greet
+  nodes/actions/identify
+  nodes/actions/learn_face
```

Comment:
```suggestion
```

I feel like we should get rid of all the action stuff, as it just isn't going to be used anytime soon, and clogs up the package a bit in the meantime